safe/datastore/test/test_folder.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool by AusAid - **Clipper test suite.**
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+
+import unittest
+
+from tempfile import mkdtemp
+from os.path import join
+from qgis.core import QgsVectorLayer, QgsRasterLayer
+from PyQt4.QtCore import QDir
+
+from safe.test.utilities import (
+    get_qgis_app,
+    standard_data_path)
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+from safe.datastore.folder import Folder
+
+
+class TestFolder(unittest.TestCase):
+    """Test the folder datastore."""
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_folder_datastore(self):
+        """Test if we can store shapefiles."""
+        path = QDir(mkdtemp())
+        ds = Folder(path)
```

Comment:
My original point is the `ds` (the variable name) should be something clearer, like `data_store`.